features/issues/components/issue-filters/issue-filters.module.scss

```diff
@@ -0,0 +1,32 @@
+@use "@styles/color";
+@use "@styles/space";
+@use "@styles/font";
+@use "@styles/breakpoint";
+
+.container {
+  display: flex;
+  flex-flow: column-reverse wrap-reverse;
+  gap: space.$s4;
+  margin-bottom: space.$s6;
+
+  @media (min-width: breakpoint.$desktop) {
+    flex-direction: row;
+  }
+
+  .filters {
+    @media (min-width: breakpoint.$desktop-large) {
+      justify-content: flex-end;
+    }
+
+    @media (min-width: breakpoint.$desktop) {
+      flex-direction: row;
+    }
+
+    flex-grow: 1;
+
+    // width: 100%;
+    display: flex;
+    flex-direction: column;
+    gap: space.$s4;
```

Comment:
I think you could move this outside the `.container` class. Or is there a specific reason why you nest it here?